pom.xml

```diff
@@ -178,22 +178,20 @@
             <artifactId>google-cloud-secretmanager</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>com.azure</groupId>
-            <artifactId>azure-security-keyvault-secrets</artifactId>
-            <version>${azure.keyvault.version}</version>
-        </dependency>
 
         <dependency>
-            <groupId>com.azure</groupId>
-            <artifactId>azure-identity</artifactId>
-            <version>${azure.identity.version}</version>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>spring-cloud-azure-dependencies</artifactId>
+            <version>${com.azure.spring.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
         </dependency>
```

Comment:
This should be in the \<dependencyManagement> section. https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html#bill-of-material-bom.

Reply:
Thank you @saragluna ! Got this one fixed :) .